search/Binary Search.cpp

```diff
@@ -11,7 +11,7 @@ l = m + 1;
 }
 return -1;
 }
-int main(int argc, char const *argv[]) {
+int main() {
```

Comment:
Why remove?

Reply:
I again add this in last commit before that I did not able to find argc variable and thought might be it give error.
Then I again add this in new commit.